.vscode/launch.json

```diff
@@ -256,20 +256,6 @@
       ],
       "protocol": "inspector"
     },
-    {
-      "type": "node",
-      "request": "launch",
-      "name": "Karma",
-      "args": [
-        "${workspaceFolder}/bin/stencil",
-        "build",
-        "--max-workers=0",
-        "--dev",
-        "--config",
-        "${workspaceFolder}/test/karma/stencil.config.ts"
-      ],
-      "protocol": "inspector"
-    },
```

Comment:
I think we should rm the configuration after this deleted one as well - the Karma Prerender test has moved as well

Reply:
indeed we should! missed that one